gh-actions/e2e/action.yaml

```
@@ -12,6 +12,18 @@ inputs:
     description: 'Working directory to run in'
     required: false
     default: '.'
+  focus:
+    description: '-ginkgo.focus argument to passed to e2e tests'
```

```suggestion
    description: '-ginkgo.focus argument to passed to E2E tests'
```

Nit: I think we capitalize it consistently in this context, so let's continue doing so.